wf/en-us/content/topics/workflows/connector-reference/azuread/actions/readgroup.htm

```diff
@@ -595,13 +600,13 @@ <h2>Output</h2>
                                                                     <td class="TableStyle-standard-horizonal-borders-only-BodyE-Column1-Body1">
                                                                         <ul>
                                                                             <li>
-                                                                                <p><span class="code">true</span> if this group is synced from an on-premises directory</p>
+                                                                                <p><span class="state">true</span> if this group is synced from an on-premises directory</p>
                                                                             </li>
                                                                             <li>
-                                                                                <p> <span class="code">false</span> if this group was originally synced from an on-premises directory but is no longer synced</p>
+                                                                                <p><span class="state">false</span> if this group was originally synced from an on-premises directory but is no longer synced</p>
                                                                             </li>
                                                                             <li>
-                                                                                <p><span class="code">null</span> if this object has never been synced from an on-premises directory (default).</p>
+                                                                                <p><span class="state">null</span> if this object has never been synced from an on-premises directory (default).</p>

```

Comment:
```suggestion
                                                                                <p><span class="state">null</span> if this object has never been synced from an on-premises directory (default)</p>

```

Reply:
I'll make these changes in the source repo.